source/gather_info.sh

```diff
@@ -287,7 +294,18 @@ cat << EOB
         "text": {
             "copy": "https://www.alfredapp.com/ (text here to copy)",
             "largetype": "https://www.alfredapp.com/ (text here for large type)"
-        }
+        },
+    },{
+        "type": "default",
+        "uid": "acct-status",
+        "title": "Local Admin Status",
+        "subtitle": "$LOCALADMIN",
+        "arg": "x",
```

Comment:
arg is what goes to the next step in the alfred workflow, idk what'd be useful here though. 
```suggestion
        "arg": "$LOCALADMIN",
```